development/tools/cmd/rendertemplates/main.go

```diff
@@ -128,6 +129,7 @@ func renderTemplate(basePath string, templateConfig *rt.TemplateConfig, config *
 	return nil
 }
 
+// renderFileFromTemplate renders template to .yaml file, based on the data passed to the template
```

Comment:
This comment and comment about renderTemplate function are almost the same. Whats? the difference then?

Reply:
I've changed the comments
renderTemplate loads the template and calls renderFileFromTemplate
renderFileFromTemplate renders the template to a file